python/ray/serve/_private/benchmarks/streaming/streaming_grpc_throughput.py

```diff
@@ -0,0 +1,215 @@
+import asyncio
+import logging
+import time
+from concurrent import futures
+from tempfile import TemporaryDirectory
+
+import click
+import grpc
+import ray
+
+from ray.serve._private.benchmarks.streaming._grpc import (
+    test_server_pb2,
+    test_server_pb2_grpc,
+)
+from ray.serve._private.benchmarks.streaming._grpc.grpc_server import TestGRPCServer
+from ray.serve._private.benchmarks.streaming.common import Caller, IOMode, GRPC_DEBUG_RUNTIME_ENV, Endpoint
+
+
+# @ray.remote(runtime_env=GRPC_DEBUG_RUNTIME_ENV)
+@ray.remote
+class EndpointActor:
+    async def __init__(self, tokens_per_request, socket_type, tempdir):
+        # Switch off logging to minimize its impact
+        logging.getLogger("ray").setLevel(logging.WARNING)
+        logging.getLogger("ray.serve").setLevel(logging.WARNING)
+
+        self.server = await self.start_server(tokens_per_request, socket_type, tempdir)
+
+        print("gRPC server started!")
+
+    @staticmethod
+    async def start_server(tokens_per_request, socket_type, tempdir):
+        server = grpc.aio.server(futures.ThreadPoolExecutor(max_workers=1))
```

Comment:
is it intended that this is not using the supported path for gRPC on Ray Serve? for regular old gRPC benchmarks we could just look up the public numbers for a rough idea of the comparison

Reply:
Yeah, the goal of this one was to specifically benchmark server/client streaming of the native gRPC

Reply:
got it. perhaps try to make that clear in a header comment on the file else it might confusing why this is here

at some point we should also add a corresponding benchmark that goes through the Serve gRPC path (including hitting the proxy)